package.json

```diff
@@ -110,6 +111,7 @@
     "semver": "^5.0.3",
     "semver-regex": "^1.0.0",
     "shelljs": "^0.6.0",
+    "tabtab": "https://registry.npmjs.org/tabtab/-/tabtab-2.2.2.tgz",
```

Comment:
🤔 Is there any way we can get this to be `"tabtab": "^2.2.2"`?

I just installed it via `npm` and get the version number back rather than a path to the `.tgz` file.

I'm more in favor to have a semver definition here rather than a path to a `.tgz` file.